<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sala</string>
  </property>
  <widget class="QLabel" name="labelCapacidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacidade:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCapacidade">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
